microservices/nris_api/openshift/_python36_oracle.dc.json

```diff
@@ -0,0 +1,444 @@
+    },
+    {
+      "name": "CPU_REQUEST",
+      "value": "150m"
+    },
+    {
+      "name": "MEMORY_REQUEST",
+      "value": "768Mi"
+    },
+    {
+      "name": "REPLICA_MIN",
+      "value": "1"
+    },
+    {
+      "name": "REPLICA_MAX",
+      "value": "1"
+    },
+    {
+      "name": "MDS_ELASTIC_SECRET",
+      "required": false,
+      "value": "template.mds-elastic-secret"
+    },
+    {
+      "name": "ELASTIC_ENABLED",
+      "value": "0"
+    },
+    {
+      "name": "ELASTIC_SERVICE_NAME",
+      "value": "MDS Dev"
+    },
+    {
+      "name": "REDIS_CONFIG_NAME",
+      "required": false,
+      "value": "template.mds-redis-secret"
+    },
+    {
+      "name": "ENVIRONMENT_NAME",
+      "displayName": "Environment Name (Environment Id)",
+      "description": "The name or Id of the environment.  This variable is used by the webhook integration to identify the environment in which the backup notifications originate.",
+      "required": false,
+      "value": "dev"
+    }
+  ],
+  "objects": [
+    {
+      "kind": "ImageStream",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}",
+        "creationTimestamp": null,
+        "labels": {
+          "shared": "true"
+        },
+        "annotations": {
+          "description": "Keeps track of changes in the application image"
+        }
+      },
+      "spec": {
+        "lookupPolicy": {
+          "local": false
+        },
+        "tags": []
+      }
+    },
+    {
+      "kind": "DeploymentConfig",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}${SUFFIX}",
+        "creationTimestamp": null,
+        "labels": null,
+        "annotations": {
+          "description": "Defines how to deploy the application server"
+        }
+      },
+      "spec": {
+        "strategy": {
+          "type": "Rolling",
+          "rollingParams": {
+            "post": {
+              "failurePolicy": "ignore",
+              "execNewPod": {
+                "containerName": "app",
+                "command": ["flask", "db upgrade"]
+              }
+            },
+            "maxSurge": "50%",
+            "maxUnavailable": 0,
+            "timeoutSeconds": 1200
+          },
+          "resources": {
+            "limits": {
+              "cpu": "250m",
+              "memory": "750Mi"
+            },
+            "requests": {
+              "cpu": "200m",
+              "memory": "500Mi"
+            }
+          },
+          "activeDeadlineSeconds": 21600
+        },
+        "triggers": [
+          {
+            "type": "ImageChange",
+            "imageChangeParams": {
+              "automatic": true,
+              "containerNames": ["app"],
+              "from": {
+                "kind": "ImageStreamTag",
+                "name": "${NAME}:${VERSION}"
+              }
+            }
+          },
+          {
+            "type": "ConfigChange"
+          }
+        ],
+        "replicas": "${{REPLICA_MIN}}",
```

Comment:
Just in case you're curious, the double {{ bracket templates input as a number whereas the single { bracket templates as a string.